Description: accept newer minor version of crate autible-serde
Author: Jonas Smedegaard <dr@jones.dk>
Last-Update: 2026-04-04
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/crates/wasm-metadata/Cargo.toml
+++ b/crates/wasm-metadata/Cargo.toml
@@ -21,7 +21,7 @@
 url = { workspace = true, optional = true }
 wasm-encoder = { workspace = true, features = ['std', 'component-model'] }
 wasmparser = { workspace = true, features = ['std', 'component-model', 'hash-collections'] }
-auditable-serde = { version = "0.8.0", optional = true }
+auditable-serde = { version = ">= 0.8.0, <= 0.9", optional = true }
 flate2 = { version = "1.1.0", optional = true }
 
 [features]
